### PR TITLE
add streaming auto test

### PR DESCRIPTION
**Pull request by aqtjin**
- [ ] streaming text classification

### CODE REVIEW

File: pyzoo/zoo/examples/streaming/textclassification/streaming_text_classification.py [hunk 1]
```diff
@@ -82,9 +85,7 @@ def predict(record):
         predicts = predict_set.get_predicts().collect()
         print("Probability distributions of top-5:")
         for p in predicts:
-            (uri, probs) = p
-            print("Predictions for " + uri + ": ")
-            for k, v in sorted(enumerate(probs[0]), key=lambda x: x[1])[:5]:
+            for k, v in sorted(enumerate(p[0]), key=lambda x: x[1])[:5]:
```
Comment (hkvision) [inline pyzoo/zoo/examples/streaming/textclassification/streaming_text_classification.py]: Why changing this?

File: pyzoo/zoo/examples/run-example-tests.sh [hunk 2]
```diff
@@ -568,3 +600,4 @@ echo "#10 pytorch time used: $time10 seconds"
 echo "#11 openvino time used: $time11 seconds"
 echo "#12 vnni/openvino time used: $time12 seconds"
 echo "#13 streaming Object Detection time used: $time13 seconds"
+echo "#14 streaming text classification time used: $time14 seconds"
```
Comment (hkvision) [inline pyzoo/zoo/examples/run-example-tests.sh]: new line

File: pyzoo/zoo/examples/run-example-tests.sh [hunk 3]
```diff
@@ -554,6 +554,38 @@ ${ANALYTICS_ZOO_HOME}/bin/spark-submit-python-with-zoo.sh \
...
+    echo "----Find-----"
+    kill -9 $(ps -ef | grep streaming_text_classification | grep -v grep |awk '{print $2}')
+    rm 1.log
+    sleep 1s
+    break
+fi
+done
+now=$(date "+%s")
+time14=$((now-start))
+
```
Comment (hkvision) [inline pyzoo/zoo/examples/run-example-tests.sh]: remove this empty line

**Comment by hkvision** [comment]
jenkins?

File: pyzoo/zoo/examples/streaming/textclassification/streaming_text_classification.py (hunk 1 above)
Comment (aqtjin) [reply pyzoo/zoo/examples/streaming/textclassification/streaming_text_classification.py]: p only includes probs

Comment (hkvision) [reply pyzoo/zoo/examples/streaming/textclassification/streaming_text_classification.py]: No, p is a tuple of uri and probs. Are you using the latest code in master?

Comment (aqtjin) [reply pyzoo/zoo/examples/streaming/textclassification/streaming_text_classification.py]: yes. I print out p and it only has probs.

Comment (hkvision) [reply pyzoo/zoo/examples/streaming/textclassification/streaming_text_classification.py]: @qiyuangong

**Pull request merged**